svg.panzoom.js

```diff
@@ -43,20 +41,20 @@ SVG.extend(SVG.Doc, SVG.Nested, {
     var pinchZoomStop = function(ev) {
       ev.preventDefault()
       zoomInProgress = false
-      
+
       SVG.off(document,'touchmove', pinchZoom)
       SVG.off(document,'touchend', pinchZoomStop)
       this.on('touchstart', pinchZoomStart)
-      
+
       if(ev.touches.length > 0) panStart.call(this, ev)
     }
 
     var pinchZoom = function(ev) {
       ev.preventDefault()
       var currentTouches = normalizeEvent(ev)
 
-      var lastDelta = Math.sqrt((lastTouches[0].clientX - lastTouches[1].clientX)**2 + (lastTouches[0].clientY - lastTouches[1].clientY)**2)
-      var currentDelta = Math.sqrt((currentTouches[0].clientX - currentTouches[1].clientX)**2 + (currentTouches[0].clientY - currentTouches[1].clientY)**2)
+      var lastDelta = Math.sqrt( Math.pow(lastTouches[0].clientX - lastTouches[1].clientX, 2) + Math.pow(lastTouches[0].clientY - lastTouches[1].clientY, 2) )
```

Comment:
Mh we might check if lastTouches[0] is always defined at this point to handle this error.
But generally there should *always* be a last touch because `pinchZoomStart` takes care of that... weired